cmake/third_party/openfst.cmake

```diff
@@ -14,12 +14,74 @@ if(NOT openfst_POPULATED)
 
     add_subdirectory(${openfst_SOURCE_DIR} ${openfst_BINARY_DIR})
 
-    install(DIRECTORY ${openfst_SOURCE_DIR}/src/include/ DESTINATION include/
-            FILES_MATCHING PATTERN "*.h")
-
-    install(TARGETS fst
-            EXPORT kaldi-targets
-            ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-            LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-            RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+    if(CONDA_ROOT)
+
+        if(MSVC)
+            install(TARGETS fstarcsort fstclosure fstcompile fstcompose fstconcat fstconnect fstconvert fstdeterminize
```

Comment:
So this was something I added due to the differences between OpenFst between Windows and other systems.  Conda has OpenFst-1.8.2 for Linux and Mac, but none for Windows (and Kaldi still depends on 1.7.X, right?).  The ideal would be to have Kaldi depend on the latest OpenFst and have OpenFst available on conda for all platforms.  I talked to Kyle Gorman back in December about porting the build code over to Cmake, but I haven't had the bandwidth to tackle that.  

Might be easier to simply use an updated version of your windows fork on conda's windows build.  From looking over the past discussions of upgrading to 1.8 last year, I got the impression that it would be a larger task.

Reply:
I started having a go at this in https://github.com/conda-forge/openfst-feedstock/pull/27

Reply:
I will be working on a support for FST 1.8+ in Kaldi, but no ETA ATM, sorry.